docs/docs/libraries/esp32-lcd-library.md

```diff
@@ -7,104 +7,100 @@ tags: ["OXRS-IO-LCD-ESP32-LIB", "TAG2", "TAG3"]
 > SKU: OXRS-IO-LCD-ESP32-LIB
 
 ## Introduction
-Shoulder picanha tongue cow kevin. Pig alcatra chicken jowl, turducken flank buffalo bresaola burgdoggen ground round venison shoulder pancetta. Sirloin chislic beef ribs turkey biltong. Ground round fatback frankfurter meatball, venison jowl capicola ham hock meatloaf. Buffalo boudin meatball ball tip spare ribs. Venison cow pig shank pastrami, chislic ham. Beef ribs hamburger sirloin brisket, capicola drumstick cupim sausage shank burgdoggen swine.
+This library serves a common status display for OXRS compatible controller with a LCD.
 
-Cupim drumstick swine venison short loin capicola bacon picanha spare ribs tongue burgdoggen pork chop cow ham. Shank brisket ball tip tri-tip cow jerky shoulder chicken, corned beef alcatra filet mignon. Biltong jerky picanha ham hock swine. Ham hock shank corned beef, porchetta strip steak venison sirloin turducken hamburger ham rump jerky. Frankfurter pork tri-tip beef ribs meatloaf picanha pastrami doner burgdoggen.
+![Image Alt Text](/images/LCD-ScreenShots.jpg)
 
----
+The screen shots above show examples for 
+  - State Monitor     (input only, 128 inputs, switches, buttons, ...) 
+  - State Controller  (output only, 128 outpus, relais)
+  - Smoke Detector    (input and output, 16 inputs, 32 outputs, special use case smoke detector)
+
+For most of the fields the displayed information is the same for all three different I/O-boards, except the port animation is dedicated to the specific I/O.
 
-### How does it work?
-Shoulder picanha tongue cow kevin. Pig alcatra chicken jowl, turducken flank buffalo bresaola burgdoggen ground round venison shoulder pancetta. Sirloin chislic beef ribs turkey biltong. Ground round fatback frankfurter meatball, venison jowl capicola ham hock meatloaf. 
+The header shows the maker logo together with FW related information (details on the logo are below). 
 
-Buffalo boudin meatball ball tip spare ribs. Venison cow pig shank pastrami, chislic ham. Beef ribs hamburger sirloin brisket, capicola drumstick cupim sausage shank burgdoggen swine.
+The second block shows 
+  - the DHCP supplied IP-address. The virtual LED to the left shows the ethernet link status up/down in green/red. This information is self discovered by the lib.
+  - the MAC address of the ethernet controller. Self discovered by the lib.
+  - the wild card MQTT topic for publish and subscribe. The two virtual LEDs to the left show MQTT activity (RX top, TX bottom) Flash when message is received or sent. Self discovered by the lib.
+  - the temperature measured by a sensor on the RACK32 controller. This has to be supplied by the FW with suitable repeat frequency.Can be any temperature.
```

Comment:
Make this more generic with a note that it is used by the RACK32 controller for on-board temp monitoring. Maybe mention the lib API call to set the temp to display in case another maker wishes to use it for something else?